plugins/inputs/redis_sentinel/redis_sentinel_test.go

```diff
@@ -312,12 +346,35 @@ func TestRedisSentinelInfoAll(t *testing.T) {
 
 	rdr := bufio.NewReader(bytes.NewReader(sentinelInfoResponse))
 
-	sentinelTags, sentinelFields, sentinalErr := convertSentinelInfoOutput(globalTags, rdr)
-	require.NoErrorf(t, sentinalErr, "failed converting output: %v", sentinalErr)
+	sentinelTags, sentinelFields, sentinelErr := convertSentinelInfoOutput(globalTags, rdr)
+	require.NoErrorf(t, sentinelErr, "failed converting output: %v", sentinelErr)
 
 	actualMetrics := []telegraf.Metric{
 		testutil.MustMetric(measurementSentinel, sentinelTags, sentinelFields, now),
 	}
 
 	testutil.RequireMetricsEqual(t, expectedMetrics, actualMetrics)
 }
+
+func createRedisContainer() testutil.Container {
+	return testutil.Container{
+		Image:    "redis:7.0-alpine",
+		Name:     "telegraf-test-redis-sentinel-redis",
+		Networks: []string{networkName},
+		WaitingFor: wait.ForAll(
+			wait.ForExposedPort(),
+		),
```

Comment:
Can you also have this wait for the log message "Ready to accept connections" as well? I like to see both a port + log. Similar to what you already do for the sentinels.

Reply:
@powersj Done. I added the additional wait condition in https://github.com/influxdata/telegraf/pull/12229/commits/79c95e8dd4007780f01adfe1b9a01afb8f8e73d4.